quickstart.md

```diff
@@ -0,0 +1,154 @@
+
+```
+GetListingsIntent what homes are available in {query}
+GetListingsIntent homes in {query}
+GetListingsIntent {query} homes available
+GetListingsIntent {query} homes for sale
+GetListingsIntent what homes are for sale in {query}
+
+CustomHelpIntent help
+CustomHelpIntent help me
+CustomHelpIntent what can I ask you
+CustomHelpIntent get help
+CustomHelpIntent to help
+CustomHelpIntent to help me
+CustomHelpIntent what commands can I ask
+CustomHelpIntent what commands can I say
+CustomHelpIntent what can I do
+CustomHelpIntent what can I use this for
+CustomHelpIntent what questions can I ask
+CustomHelpIntent what can you do
+CustomHelpIntent what do you do
+CustomHelpIntent how do I use you
+CustomHelpIntent how can I use you
+CustomHelpIntent what can you tell me
+```
+
+Here we have the name of the intent on the left, the utterance on the right, and the slot in curly braces. You can find more information on sample utterances in [the Amazon documentation](https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/defining-the-voice-interface#h2_sample_utterances).
+
+Save that, and on the next screen select the radio button for **AWS Lambda ARN (Amazon Resource Name)**, select the region you selected for your Lambda function, and paste in the ARN from your Lambda function. The ARN can be found in the upper-righthand corner on the Lambda function page.
+
+Save this, we'll be coming back to it later.
+
+## Uploading and testing our code
+
+Write your code using the Algolia Alexa adapter, making sure you've installed the `algoliasearch-alexa` npm module and that you have your code in an index.js file, exporting the returned function result as the only export of that file. For example:
+
+```javascript
+const algoliaAlexaAdapter = require('algoliasearch-alexa').default;
+
+const voiceSearch = algoliaAlexaAdapter({
+  algoliaAppId: 'applicationId',
+  algoliaApiKey: 'publicSearchKey',
+  defaultIndexName: 'listings',
+  handlers: {
+    onLaunch(launchRequest, session, response) {
+      this.emit(':tell', 'Welcome to the skill!')
+    },
+    GetListingsIntent: {
+      answerWith: function (data) {
+        if(data.results.length) {
+          this.emit(':tell', `There were ${data.results.hits.length} listings found.`);
+        } else {
+          this.emit(':tell', 'We could find no listings. Please try again.');
+        }
+      },
+    },
+    CustomHelpIntent: function (intent, session, response) {
+      const speechOutput = 'Find one of thousands of listings from the Listing Store, powered by Algolia.';
+      this.emit(':ask', speechOutput);
+    },
+  },
+});
+
+module.exports = voiceSearch;
+```
+
+Notice the synchronicity between our handler name and our intent schema. (`GetListingsIntent` in both.)
+
+Next, zip up both the `index.js` file and `node_modules` directory *without* zipping them into a directory themselves.

```

Comment:
Nah, that's a very good point. Usually when you zip something from the GUI it gets put into a directory of its own that then gets zipped together. We want to avoid that. There's a better way to phrase this for sure.